### DISCUSSION TITLE
Activating extension 'undefined_publisher.queryneptunegraph' failed: Unexpected end of input.

### DISCUSSION THREAD

**Discussion by AndreaNassisi** [microsoft/vscode-discussions#896] [Extension Development QnA]
**I can no longer debug my extension**, I'm now getting:
```Activating extension 'undefined_publisher.queryneptunegraph' failed: Unexpected end of input.```

1/ I created a new extension using the template
2/ was working
3/ I ran ```vsce package``` to create a VSIX. I then install the VSIX in a remote machine to do more testing.
4/ I came back to the development project of the extensions, made changes to the code.
5/ Debugging the code extension no longer works.

Note: I do not have a registered publisher account yet.

**Comment by r3m0t** [comment]
`vsce package` might clear the output folder of `tsc` to do a clean build. So I would restart the build task `npm: watch` and check the js file mentioned in `package.json` is present. As well, check all channels in the `Output` panel for more detailed error messages.

**Reply by AndreaNassisi** [reply]
Thank you r3m0t! I'm using the JavaScript template, not the TypeScript. Maybe that's the issue. I will start a new project with TypeScript and see...
Thank you again.

**Reply by AndreaNassisi** [reply]
Found the issue, a function was missing a closure }. Strange error ...

**Comment by AndreaNassisi** [comment]
Solved. Missing a function closure.
